assets/scss/core/_mixins.scss

```diff
@@ -60,35 +60,46 @@
 
 @mixin font($preset) {
     @if $preset == title {
-        font: bold 1.75rem /#{(32 / 28)} $font-family-sans-serif;
-        font-feature-settings: "dlig" 1;
+        font-size: 1.75rem;
+        line-height: #{(32 / 28)};
```

Comment:
the `#{}` wrapper should bot be necessary in a separate statement, but i guess it does not matter much.

Reply:
Actually it didn't work without it, `32 / 28` being a valid CSS syntax that Sass don’t interpret apparently 😅